harmonic/install_windows_src.md

```diff
@@ -4,14 +4,14 @@ WARNING: Current Windows support is experimental.
 
 # Source Installation on Windows 10 or 11
 
-Command line tools, DART physics engine, and GUI capabilities are
-not currently supported in Windows. These functionalities correspond to the currently
-building packages `gz-tools`, `gz-physics`, and `gz-gui`, respectively.
+OGRE2 rendering capabilities are not currently supported in Windows, and Gazebo GUI
+works in a limited fashion. These functionalities correspond to the currently
+building packages `gz-rendering` and `gz-sim`, respectively. The packages will build,
+but you can expect runtime failures when using their functionalities.
 
-**Note**
-
-You will still be able to use `TPE` as a physics engine
-(see [here](https://gazebosim.org/api/physics/2.2/physicsplugin.html) for more information on `TPE`).
+> [!NOTE]
```

Comment:
This `[!NOTE]` is a GitHub Markdown format that will not work on the rendered webpage. The `docs` repo is meant to be deployed to the Gazebo website, so the formatting has to target the website, not GitHub. You can see the rendered page here https://gazebosim.org/docs/harmonic/install_windows_src

For fancy stuff like warning bars (see the first line in this file), we have to add CSS to the website frontend for them to work. For example, the yellow warning bar format was added here: https://github.com/gazebo-web/gazebosim-web-frontend/pull/12

For now, we need to use the original format here. Otherwise it just looks like this when I test the PR locally:
![Screenshot from 2023-09-15 20-16-04](https://github.com/gazebosim/docs/assets/7608908/c188ddad-955f-4259-944a-d1f8b0a26908)
```suggestion
> **NOTE**
```